packages/cli/package.json

```diff
@@ -54,11 +54,11 @@
     "strip-ansi": "6.0",
     "text-table": "0.2",
     "tslib": "^2.3.0",
-    "yargs": "17.2.1"
+    "yargs": "17.3.1"
```

Comment:
Was this a regression with yargs or something?

Reply:
nope, it wasn't, I just updated it while I was at it.